src/components/forms/FieldWrapper.tsx

```diff
@@ -126,26 +135,41 @@ function FieldWrapper(props, ref) {
       aria-label={label}
       {...labelProps}
     >
-      {extra ? (
-        <Grid placeContent="baseline space-between" flow="column">
-          <div>{label}</div>
-          <Text preset="t3">{extra}</Text>
-        </Grid>
-      ) : (
-        label
-      )}
-      {tooltip ? (
-        <>
-          &nbsp;
-          <TooltipProvider
-            title={tooltip}
-            activeWrap
-            width="initial max-content 40x"
+      <Flex flex="1 0 100%">
+        {extra ? (
+          <Grid
+            placeContent="baseline space-between"
+            flow="column"
+            width="100%"
           >
-            <InfoCircleOutlined style={{ color: 'var(--primary-color)' }} />
-          </TooltipProvider>
-        </>
-      ) : null}
+            <Space placeItems="baseline" gap="0.5x">
+              <div>{labelPrefix}</div>
+              <div>{label}</div>
+              <div>{labelSuffix}</div>
+            </Space>
+
+            <Text preset="t3">{extra}</Text>
+          </Grid>
+        ) : (
+          <Space placeItems="baseline" gap="0.5x">
+            <div>{labelPrefix}</div>
+            <div>{label}</div>
+            <div>{labelSuffix}</div>
+          </Space>
+        )}
+        {tooltip ? (
```

Comment:
Looks like `tooltip` is not compatible with `extra` and `labelSuffix` props. Can you fix it? It should go right after the label text.

Reply:
Am I right that this should be right between the `label` and the `labelSuffix` ?

Reply:
Thanks! fixed.